ibtk/config/config.h.tmp

```diff
@@ -66,4 +66,9 @@ _Pragma("GCC diagnostic ignored \"-Wunneeded-internal-declaration\"")
 #include <ibtk/compiler_hints.h>
 #include <ibtk/ibtk_utilities.h>
 
+// Tell boost to get with the program:
```

Comment:
```suggestion
// Disable std::auto_ptr inside our bundled copy of boost:
```

Reply:
Unfortunately this macro does much more than just disable `auto_ptr`. We need it here to disable the use of `unary_function`.

Reply:
How about `// Disable features which were deprecated in C++11 and removed in C++17 in boost:`

Reply:
Isn't that what I wrote? 😁

Reply:
Technically but I didn't know that it was for the other deprecated then removed functions 🙈

Reply:
I think this might be a MSVC flag that Boost has adopted...?